lib/jxl/dec_external_image.h

```diff
@@ -35,6 +35,7 @@ namespace jxl {
 // xsize and ysize.
 Status ConvertToExternal(const jxl::ImageBundle& ib, size_t bits_per_sample,
```

Comment:
Please document what's the meaning of want_extra_channel and what are the restrictions about calling this function with want_extra_channel=true.

Reply:
Done